_posts/blog/2015-11-08-gtac.md

```diff
@@ -15,13 +15,87 @@ date: 2015-11-08T15:39:55-04:00
+</picture>
+</center>
+
+Let's get down to business. There were several great talks. The scope of topics ranged from frameworks for mobile testing through test processes up to massive test racks for Google Chromecast Sticks and roboters with highspeed cameras. The following selection of talks does not claim to be complete. But this were the sessions that impressed me the most.
+
+### The Uber Challenge of Cross-Application/Cross-Device Testing
+<iframe width="560" height="315" src="https://www.youtube.com/embed/p6gsssppeT0" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1vYXhkvgLKun72Ix91LQDDWZQdcY5VOBqKVvI1Y6riYo/pub">slides</a>
+
+Uber presented a testing-framework that they implemented to allow ui based testing of interconnected (and cross plattform) apps like their driver and passenger app. If you are in the same situation and try to automate a test that needs communication between two apps (messenger, games, etc) give it try!
+
+### Robot Assisted Test Automation
+<iframe width="560" height="315" src="https://www.youtube.com/embed/oQRrk7S9sUE" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1DxEX7d3UZKFv-ekHzt1-eRCWtlR8uAyXt6ra1B14IGQ/pub">slides</a>
+
+Fanzy robot hands and fingers combined with a high speed camera that allow reproducible tests on every touchscreen. There main purpose is to test the accuracy and speed of displays and the devices in general. But there are so cool, I would love to have one of these things at work.
+
+### Chromecast Test Automation
+<iframe width="560" height="315" src="https://www.youtube.com/embed/TfAVJLhb_k0?list=PLSIUOFhnxEiCWGsN9t5A-XOhRbmz54IS1" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1gBGS5UFyyR0mcwg990I1YDOFQwAhL-nmonNPGvGYnlk/pub">slides</a>
+
+Did you ever asked yourself how Google tests hardware like the chromecasts? Well this talk gives you the answer. In a Google way - means in a massive scale with a lot of intelligence behind it.
+
+### Your Tests Aren't Flaky
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hmk1h40shaE?list=PLSIUOFhnxEiCWGsN9t5A-XOhRbmz54IS1" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1L9hGYqCAgjZyXE9ch4Toh4ziuYYkB2OiMCdFpgfTko0/pub?pageId=109506508449676915259&slide=id.gd8d3f5279_0_0">slides</a>
+
+I'm pretty sure you know sentences like this: "Ahh - your test is just flaky, just re-run it. If it is green at the second run, everything is fine!" Well this talk from Alister Scott tells you a different story. Something like flaky tests doesn't exist - if your test results are non-deterministic, it could actually be the fault of the system under test. If you want more after watching his tals - he has a highly recommandable <a href="http://watirmelon.com/">blog (watirmelon.com)</a> that gives a lot tips about test automation on general.
```

Comment:
If it's green in the second run, ....
Wanna know more after watching his talk - I highly recommend his blog.... he gives a lot of advice on test automation in general